__tests__/app.test.js

```diff
@@ -73,6 +73,82 @@ describe("App", () => {
           });
         });
     });
+    // Query tests
+    test("200: Responds with the array of review objects, all of which only have the category of 'dexterity'", () => {
+      return request(app)
+        .get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews.length).toBeGreaterThanOrEqual(1);
+          reviews.forEach((review) => {
+            expect(review).toHaveProperty("category", "dexterity");
+          });
+        });
+    });
+    test("200: Responds with each array of review objects sorted by each valid sort_by query (using default order)", () => {
+      const requestFunc = (query) => {
+        return request(app)
+          .get(`/api/reviews?sort_by=${query}`)
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toBeSortedBy(query, { descending: true });
+          });
+      };
+      return Promise.all([
+        requestFunc("review_id"),
+        requestFunc("title"),
+        requestFunc("owner"),
+        requestFunc("category"),
+        requestFunc("created_at"),
+        requestFunc("votes"),
+        requestFunc("comment_count"),
+      ]);
+    });
+    test("200: Responds with the array of review objects ordered by ascending (using default sort_by)", () => {
+      return request(app)
+        .get("/api/reviews?order=asc")
```

Comment:
If it's a default, you shouldn't put the query in the request. We want to see what happens when the query isn't passed.